common/src/test/java/org/apache/falcon/entity/parser/FeedEntityParserTest.java

```diff
@@ -1160,6 +1160,24 @@ public void testImportFeedSqoopInvalid() throws Exception {
         Assert.fail("ValidationException should have been thrown");
     }
 
+    @Test (expectedExceptions = {ValidationException.class, FalconException.class})
+    public void testImportFeedWithNoTimePartition() throws Exception {
+
+        InputStream feedStream = this.getClass()
+                .getResourceAsStream("/config/feed/feed-import-no-timepartition-0.1.xml");
+        parser.parseAndValidate(feedStream);
+        Assert.fail("ValidationException should have been thrown");
+    }
+
+    @Test (expectedExceptions = {ValidationException.class, FalconException.class})
```

Comment:
Shouldn't this be just ValidationException.class. Using two exceptions make the test a bit indeterministic and may pass in other cases as well.